components/Workspaces/WorkspaceIssuesTable.tsx

```diff
@@ -0,0 +1,308 @@
+import {
```

Comment:
This is pretty much the same thing as the PRs table feature, aside from the different data rendered.